File: pkg/pwrgrep/testdata/fixtures/java/SwallowedErrors.java
public class SwallowedErrors {
    void f() {
        try {
            run();
            // ruleid: java-swallowed-errors
        } catch (Exception e) {}
        try {
            run();
            // ok: java-swallowed-errors
        } catch (RuntimeException e) {
            throw new IllegalStateException(e);
        }
    }

    void run() {}
}

File: pkg/pwrgrep/testdata/fixtures/csharp/SwallowedErrors.cs
public class SwallowedErrors
{
    void F()
    {
        try { Run(); }
        // ruleid: cs-swallowed-errors
        catch (IOException e) { }
        try { Run(); }
        // ruleid: cs-swallowed-errors
        catch { }
        try { Run(); }
        // ok: cs-swallowed-errors
        catch (Exception e) { throw new InvalidOperationException("failed", e); }
    }

    void Run() { }
}
